### DISCUSSION TITLE
Cuboid xml annotation

### DISCUSSION THREAD

**Discussion by root-dh-94** [cvat-ai/cvat#3986] [Q&A]
Hi!

We used cvat to label our images with cuboids (as 3D bounding boxes). I downloaded the annotations as an xml file and was wondering where I could find what each of the points in the file correspond to and what they actually mean (example: ybr2, xbr2, ybl2, xbl2), as I was looking to convert this file into a format that could be input into the cityscapes evaluator. 
Thank you

**Comment by bsekachev** [comment]
1 - front surface
2 - back surface

So, we have two rectangles. One in front, one in back. Each has 4 vertexes.

xtl - x top left; 
ytl - y top left;

xtr - x top right;
ytr - y top right;

xbr - x bottom right;
ybr - y bottom right;

xbl - x bottom left;
ybl - y bottom left;

xbl1, xbl2 means for example bottom left vertex of the front surface
xtr2, ytr2 means for example top right vertex of the back surface


![image](https://user-images.githubusercontent.com/40690378/145593022-23e8cfce-1e21-48ea-8078-92f70fc8509b.png)
